### ISSUE TITLE
Dependency is undefined

### ISSUE THREAD

**Issue by katima-g33k** [plbrault/weasleyjs#1]
## Problem
When registering dependencies with the same path, the ones declared before are overwritten to undefined.

## To reproduce
```javascript
// weasly.js
import Weasley from 'weasley';

const weasley = new Weasley();
export default weasley;

weasley.register('lib.Logger', () => require('./Logger'));
weasley.register('lib.Settings', () => require('./Settings'));

// Usage
import weasley from './weasley';

const Logger = weasley.container.lib.Logger;
Logger.getLogger(this.constructor.name); // Error cannot call `getLogger` of `undefined`
```

**Comment by plbrault** [comment]
Thanks for your report. I will fix this in early January.

**Comment by plbrault** [comment]
It should be fixed by release 0.1.9.
